packages/address-consts/package.json

```diff
@@ -6,10 +6,6 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "sideEffects": false,
-  "scripts": {
-    "build": "tsc --p tsconfig.build.json",
-    "prepublishOnly": "yarn run build"
```

Comment:
I'm not 100% sure about this, but I don't believe we need these `prepublishOnly` hooks anymore. sewing-kit monorepo packages don't include any of these so I presume it's not necessary.